src/elements/content-sidebar/versions/VersionsItem.js

```diff
@@ -86,9 +98,12 @@ const VersionsItem = ({
     const isDownloadable = !!is_download_available;
     const isLimited = versionCount - versionInteger >= versionLimit;
     const isRestricted = isWatermarked && !isCurrent; // Watermarked files do not support prior version preview
+    const isRetentionActive =
```

Comment:
Can we make this fit on one line?

Reply:
It becomes three lines after changing the prefix to `retention`  😂😂😂